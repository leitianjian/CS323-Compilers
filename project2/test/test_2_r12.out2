Error type 12 at Line 15: array indexing with non-integer type expression
Error type 5 at Line 15: unmatching type on both sides of assignment
