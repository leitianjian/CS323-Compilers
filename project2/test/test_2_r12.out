Error type 12 at line 15: array indexing with non-integer type expression
